src/services/script_patcher/script_patch.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <memory/pattern.hpp>
```

Comment:
Using `<` include while somewhere else `"` is used to include this same header.

I understand the usage of `<` includes for stuff that comes from "modules".